clinica/utils/input_files.py

```diff
@@ -194,12 +199,54 @@
 # T1-Volume
 
 
+def aggregator(func):
+    """If the decorated function receives iterable arguments,
+    this decorator will call the decorated function for each
+    value in the iterable and aggregate the results in a list.
+    This works only if the iterables provided have the same length.
+    Arguments lefts as non-iterable will be repeated.
```

Comment:
I had to read this multiple times to understand what it does. I believe a few exemples would be welcome.

Reply:
See my answer to your other comment below.